src/classes/api/lumedicApi.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+export default {
+    postData: (values) =>
```

Comment:
please create a validate method instead and do not have a default export, as the lumedicAPI.js file can be used for the other API call methods as well